### PR TITLE
Personalizer-Rename csproj

### PR DESCRIPTION
**Pull request by mandyshieh**

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Microsoft.Azure.CognitiveServices.Personalizer.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
-    <Version>0.8.0-preview</Version>
+    <Version>1.0.0-preview</Version>
```
Comment (yangyuan) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Microsoft.Azure.CognitiveServices.Personalizer.csproj]: Are you sure you want 1.0.0-preview?
I would recommend to stay in 0.x.0-preview...

**Review by yangyuan — CHANGES_REQUESTED** [review]
suggest a less aggressive version changes (but up to you)

Properties.cs also need to be changed.

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Microsoft.Azure.CognitiveServices.Personalizer.csproj (hunk 1 above)
Comment (mandyshieh) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Microsoft.Azure.CognitiveServices.Personalizer.csproj]: good point, changed to 0.9.0 and updated AssemblyInfo.cs too

**Review by shahabhijeet — APPROVED** [review]
Once the CI passes, this PR is good to be merged

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**